### PR TITLE
add f17-core config

### PR DESCRIPTION
**Pull request by imaginator**
initial config to connect:
- emma
- zwingli
- rathausneukoelln

and serve dhcp addresses to multiple APs (hof/street) serving neighbours and passers by.

### CODE REVIEW

File: group_vars/location_f17/networks.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# 10.36.189.0/24
+# 2001:bf7:830:6600::/56
+
+ipv6_prefix: "2001:bf7:830:b000::/56"
+
+networks:
+  - vid: 6
+    role: mesh
+    name: mesh_emma
+    prefix: 10.31.167.209/32
```
Comment (PolynomialDivision) [inline group_vars/location_f17/networks.yml:20]: You can start with `10.31.167.208/32` as ip from your mesh-pool. :)

File: group_vars/location_f17/networks.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+#	- 2001:bf7:830:b080::/57               (for announcing)
+
+# DHCP:
+# 10.36.189.0/24
+# 2001:bf7:830:6600::/56
+
+ipv6_prefix: "2001:bf7:830:b000::/56"
+
+networks:
+  - vid: 6
```
Comment (Noki) [inline group_vars/location_f17/networks.yml:17]: please use the VLAN numbering convention for new locations (https://github.com/freifunk-berlin/bbb-configs/blob/master/DEVELOPER.md#vlan-numbering-convention)

File: group_vars/location_f17/owm.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+location_nice: Falckensteinstr 17
+latitude: 52.4959365
+longitude: 13.4447219
```
Comment (PolynomialDivision) [inline group_vars/location_f17/owm.yml:4]: Are u sure your geo-coordinates are correct?
https://hopglass.berlin.freifunk.net/#!v:m;n:f17-core.olsr

File: group_vars/location_f17/snmp.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+
+snmp_devices:
+  - hostname: f17-emma
+    address: 10.31.167.195
+    snmp_profile: airos_6
+  - hostname: f17-zwingli
+    address: 10.31.167.196
+    snmp_profile: airos_8
```
[the remark this answers is not in the document]
Comment (imaginator) [reply group_vars/location_f17/snmp.yml:9]: should be fixed now.

File: group_vars/location_f17/networks.yml (hunk 1 above)
Comment (imaginator) [reply group_vars/location_f17/networks.yml]: done!

File: group_vars/location_f17/networks.yml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+# 10.31.167.192/27
+#	- 10.31.167.192/28 - 10.31.167.207/28  (management network)
+#	- 10.31.167.208/28 - 10.31.167.223/28  (for announcing)
+# 2001:bf7:830:b000::/56
+#	- 2001:bf7:830:b000::/57               (management network)
+#	- 2001:bf7:830:b080::/57               (for announcing)
+
+# DHCP:
+# 10.36.189.0/24
+# 2001:bf7:830:6600::/56
```
[the remark this answers is not in the document]
Comment (imaginator) [reply group_vars/location_f17/networks.yml]: :+1:

File: group_vars/location_f17/networks.yml (hunk 2 above)
Comment (imaginator) [reply group_vars/location_f17/networks.yml]: VLANs standardised now

File: group_vars/location_f17/owm.yml (hunk 3 above)
Comment (imaginator) [reply group_vars/location_f17/owm.yml]: fixed.

**Comment by PolynomialDivision** [comment]
Plesse, squash your commits into a single one and add a location prefix, i.e. `f17: ` :)

**Pull request closed without merging**